src/zjs_common.c

```diff
@@ -27,23 +27,49 @@ char *zjs_shorten_filepath(char *filepath)
 }
 
 #ifdef DEBUG_BUILD
+#ifdef ZJS_LINUX_BUILD
+#include <time.h>
 
+static u8_t init = 0;
+static int seconds = 0;
+#else
 static int start = 0;
+#endif
 
 int zjs_get_sec(void)
 {
+#ifdef ZJS_LINUX_BUILD
```

Comment:
I think this could be left without the ifdef. The Linux port should have `zjs_port_timer_get_uptime()` and if it doesn't then you could just move this ifdef'ed code into that function. That would at least simplify this piece.